### DISCUSSION TITLE
Carla sumo co-simulation map problem

### DISCUSSION THREAD

**Discussion by nikolatopic131** [carla-simulator/carla#7426] [Q&A]
Hello. I'm having a problem whilst running a carla sumo co-simulation. The vehicles are driving on the emptiness, slightly away from the road, as can be seen in the provided screenshot. What I've done is, I've taken the .net.xml file that is my sumo net and converted it to .xodr using netconvert, then I loaded the map in carla with config.py, started the carla server and then started the co-simulation with run_synchronization.py. Does someone know why this happens and how to fix it?
![carlasumo](https://github.com/carla-simulator/carla/assets/162899513/e5415914-d084-4799-a278-398a85c251bd)

**Comment by Sima-Ashayer** [comment]
Did you find any solution for this issue?

**Reply by nikolatopic131** [reply]
Hello. Yes, in my sumo .net.xml map file there was a location tag with a netOffset atribute. I've set it to 0 and it is fine now.

**Reply by Sima-Ashayer** [reply]
Thank you. Do you have any idea about doing this in PTV Vissim? because in PTV Vissim we only have one .inpx file.

**Reply by nikolatopic131** [reply]
Do you have the map and vehicles offset problem like I had or?

**Reply by Sima-Ashayer** [reply]
Yes, I have the exact problem you were faced, but I'm creating the map in PTV Vissim instead of Sumo.

**Reply by nikolatopic131** [reply]
Unfortunately, I am not experienced in Vissim, I only have basic knowledge. But I suppose you could check your map files, both opendrive and vissim for any offset attributes like I had. Alternatively, you could check your georeference if it is a georeferenced map, to see if the coordinates are the same. Other than that I don't really have any ideas. Perhaps it could also be a projection issue, not sure tbh.

**Reply by Sima-Ashayer** [reply]
OK, Thank you!

**Comment by Sephiroth1118** [comment]
Hi! I just had the same issue. I was running another repo which is using sumo and carla co-simulation in the Town01 map. The strange thing is when I change the value of netOffset, nothing happened. I'm confused.
